src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -17,6 +17,7 @@ class IntroductionPage extends React.Component {
         <FormTitle title="This page needs attention"/>
         <p>Equal to VA Form 21-526EZ.</p>
         <SaveInProgressIntro
+          {...this.props}
```

Comment:
Is this change still necessary?

Reply:
`<SaveInProgressIntro/>` now needs `form`, `route`, `location`, and `pageList `from `this.props` to calculate the first page, should we enumerate them instead

Reply:
Thanks @jbalboni, great idea, updated.